K2Bridge/KustoDAL/KustoDataAccess.cs

```diff
@@ -115,6 +115,41 @@ public async Task<IndexListResponseElement> GetIndexListAsync(string indexName)
             return response;
         }
 
+        /// <summary>
+        /// Executes a query to Kusto for Index List.
+        /// Searches for all tables and all functions that match the index name pattern.
+        /// </summary>
+        /// <param name="indexName">Index name pattern, e.g. "*", "orders*", "orders".</param>
+        /// <returns>A list of Indexes matching the given name pattern.</returns>
+        public async Task<ResolveIndexResponse> ResolveIndexAsync(string indexName)
```

Comment:
Can we delete GetIndexListAsync and every non needed method?
I think its the same exact code, (except one line inside the if statement)

Reply:
I will look at combining both methods into one.

Reply:
I have refactored to avoid code duplication.